tokio-reactor/src/lib.rs

```diff
@@ -2,6 +2,7 @@
 #![deny(missing_docs, missing_debug_implementations, rust_2018_idioms)]
 #![cfg_attr(test, deny(warnings))]
 #![doc(test(no_crate_inject, attr(deny(rust_2018_idioms))))]
+#![allow(clippy::trivially_copy_pass_by_ref)]
```

Comment:
Is it possible to allow this on a case by case basis instead of at the file level? I'm not sure what this is applying to.

Reply:
That makes sense. done in https://github.com/tokio-rs/tokio/pull/1335/commits/f549d8b06d1528b75e357541dde3fb5e4fd04696.